### ISSUE TITLE
Implement option for ISO 8601 timestamps

### ISSUE THREAD

**Issue by NCommander** [SoylentNews/rehash#229]
Got the suggestion on the last feedback. Should be a nice easy task for anyone who wants to pickup the codebase.

**Comment by darkfeline** [comment]
Idle thought: performance-wise, would it be a problem to allow users to provide their own strftime-formatted string?  That would remove the need for the (in my opinion) odd set of date/time formats that SN has at the moment.

**Comment by TheMightyBuzzard** [comment]
I don't *think* it would be too much of a hit. I mean we're basically already doing that just with a limited set.

**Comment by marty-b** [comment]
On 11/19/2015 10:29 AM, TheMightyBuzzard wrote:
>
> I don't /think/ it would be too much of a hit. I mean we're basically 
> already doing that just with a limited set.
>
> —
> Reply to this email directly or view it on GitHub 
> <https://github.com/SoylentNews/rehash/issues/229#issuecomment-158090215>.
>
I really like the idea of custom date/time stamp displays. Along with 
the suggested ISO8601, I'd very much like to be able to create/use 
something along the lines of: ccyymmdd_HHMMSS (DOW) where DOW ==> DayOf 
Week (e.g. WED)

Would this require adding a new field in the DB on a per-user basis to 
store their custom setting?

**Comment by TheMightyBuzzard** [comment]
Yep, that's exactly what it would require along with changes to the code. No big whoop though. Easy to do, it'd just be a little time consuming.
